core/src/main/java/org/geysermc/geyser/registry/type/GeyserMappingItem.java

```diff
@@ -42,6 +42,9 @@ public class GeyserMappingItem {
     @JsonProperty("stack_size") int stackSize = 64;
     @JsonProperty("tool_type") String toolType;
     @JsonProperty("tool_tier") String toolTier;
+    @JsonProperty("armor_type") String armorType;
```

Comment:
Make sure you update the mappings submodule to point to a commit with these properties.

Reply:
Well there isn't a commit for it on that repo yet